### PR TITLE
various python related tweaks

### PR DESCRIPTION
**Pull request by bpkroth**
A collection of minor python related tweaks while working on #66.

### CODE REVIEW

File: scripts/run-python-tests.sh [hunk 1]
```diff
@@ -15,4 +15,4 @@ pythoncmd=`getPythonCmd`
 
 # Linux filesystems are case-sensitive, so we need to tell the python unit test
 # scanner to look for upper-case files as well:
-$pythoncmd -m unittest discover --verbose --locals --failfast -s source/Mlos.Python -p "[Tt]est*.py"
+$pythoncmd -m unittest discover --verbose --locals --failfast -s source/Mlos.Python -p "[Tt]est*.py" $*
```
Comment (amueller) [inline scripts/run-python-tests.sh:18]: I don't actually know what that does.

**Review by amueller — APPROVED** [review]
My bash scripts aren't entirely on your level but looks good :)

File: scripts/run-python-tests.sh (hunk 1 above)
Comment (bpkroth) [reply scripts/run-python-tests.sh:18]: The explanation was in the commit message, but I suppose that isn't as helpful.
https://github.com/microsoft/MLOS/pull/73/commits/93eeb691b491aa33af9a7b65873f77f5a7ccb815

Basically, if you call:
```sh
./run-python-test.sh foo bar
```

Then `$*` evaluates as `foo bar`.

The main purpose was to allow more quickly rerunning just the failed unit test with `-k some_test_name` appended afterwards to more quickly iterate on fixes.

Should I document that, or do you want to leave it for a `pytest` conversion fixup (#72)?

Comment (amueller) [reply scripts/run-python-tests.sh:18]: ah, cool. I think this is fine for now and we can leave it for later.

**Pull request merged**